<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Manoj" failures="1" tests="3" name="Test" time="28.439" errors="0" timestamp="2025-01-02T21:23:08 IST">
  <testcase classname="com.divsly.testcases.TC_LoginPageTest" name="userLogin" time="15.083"/>
  <testcase classname="com.divsly.testcases.TC_LoginPageTest" name="invaliduserLogin" time="12.131"/>
  <testcase classname="com.divsly.testcases.TC_LoginPageTest" name="invalidEmailIdLogin" time="1.131">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.divsly.testcases.TC_LoginPageTest.invalidEmailIdLogin(TC_LoginPageTest.java:63)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- invalidEmailIdLogin -->
</testsuite> <!-- Test -->
